README.md

```diff
@@ -14,9 +14,345 @@ ManageIQ plugin for the Hawkular provider.
+sudo systemctl enable postgresql
+sudo systemctl start postgresql
+sudo su postgres -c "psql -c \"CREATE ROLE root SUPERUSER LOGIN PASSWORD 'smartvm'\""
+```
+
+#### OS X
+
+TODO.
+
+#### Installing a Ruby Runtime
+
+There are many options to have different ruby versions running at the same
+time, and while the Ruby versions in the repositories are usually up to date
+enough, there's often a need to test or fix stuff in different ruby versions,
+so the recommended way is to use a ruby version manager, such as
+[RVM](https://rvm.io/), [Rbenv](https://github.com/rbenv/rbenv) or
+[Chruby](https://github.com/postmodern/chruby). In this guide we'll be covering
+Rbenv.
+
+For OS X, `rbenv` and `ruby-build` are available on homebrew, and can be installed like this:
+
+```bash
+brew upgrade rbenv ruby-build
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+eval "$(rbenv init -)"
+```
+
+For Linux, it will have to be installed via default git checkout:
+
+```bash
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+cd ~/.rbenv && src/configure && make -C src
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+mkdir -p "$(rbenv root)"/plugins
+git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+eval "$(rbenv init -)"
+```
+
+After this, you can install ruby with:
+
+```bash
+rbenv install 2.4.1
+```
+
+And after that, install bundler:
+
+```bash
+gem install bundler --no-ri --no-rdoc
+```
+
+### Cloning and configuring the application
+
+The first necessary step is to fork the following repositories:
+
+* [ManageIQ Core](https://github.com/ManageIQ/manageiq)
+* [ManageIQ UI Classic](https://github.com/ManageIQ/manageiq-ui-classic)
+* [ManageIQ Schema](https://github.com/ManageIQ/manageiq-schema)
+* [ManageIQ API](https://github.com/ManageIQ/manageiq-api)
+* [ManageIQ Hawkular Provider](https://github.com/ManageIQ/manageiq-providers-hawkular)
+* [Hawkular Ruby Client](https://github.com/hawkular/hawkular-client-ruby)
+
+Then, we clone them:
+
+```bash
+mkdir ~/ManageIQ && cd ~/ManageIQ
+git clone git@github.com:<your-username>/manageiq
+git clone git@github.com:<your-username>/manageiq-ui-classic
+git clone git@github.com:<your-username>/manageiq-schema
+git clone git@github.com:<your-username>/manageiq-api
+git clone git@github.com:<your-username>/manageiq-providers-hawkular
+git clone git@github.com:<your-username>/hawkular-client-ruby
+```
+
+And then we add the upstream remotes to fetch changes on the main codebase:
+
+```bash
+cd ~/ManageIQ/manageiq
+git remote add upstream git@github.com:ManageIQ/manageiq
+cd ..
+
+cd manageiq-ui-classic
+git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic
+cd ..
+
+cd manageiq-schema
+git remote add upstream git@github.com:ManageIQ/manageiq-schema
+cd ..
+
+cd manageiq-api
+git remote add upstream git@github.com:ManageIQ/manageiq-api
+cd ..
```

Comment:
can't all cd point to the next folder already without first going to ".." ?

e.g. 
 +cd manageiq-api
 +git remote add upstream git@github.com:ManageIQ/manageiq-api
 +cd ../manageiq-providers-hawkular

Reply:
@cfcosta ^^

Reply:
Instead of cd-ing, you could also pass -C flag to simplify:

`git -C manageiq-ui-classic/  remote add upstream git@github.com:ManageIQ/manageiq-ui-classic`